Cloud security analytics guide (#2658)
SumoLogic/sumologic-documentation: docs/csa/data-lakes.md

---
id: data-lakes
title: Data Lakes
sidebar_label: Data Lakes
description: Learn about using data lakes with Cloud Security Analytics. 
---

import useBaseUrl from '@docusaurus/useBaseUrl';

> To use a data lake with Cloud Security Analytics, you must first [install a data lake](#install-a-data-lake-for-use-with-cloud-security-analytics). Cloud Security Analytics is optimized to work with Amazon Data Lake, but you can use the data lake of your choice. Ask your Sumo Logic account representative to engage the Professional Services team to guide you through setting up a data lake for use with Cloud Security Analytics.

Reviewer: ```suggestion
To use a data lake with Cloud Security Analytics, you must first [install a data lake](#install-a-data-lake-for-use-with-cloud-security-analytics). Although Cloud Security Analytics is optimized to work with Amazon Data Lake, you can use the data lake of your choice.

Ask your Sumo Logic account representative to engage the Professional Services team to guide you through setting up a data lake for use with Cloud Security Analytics.
```

## About data lakes

> A data lake is a centralized repository to store data. Data in the lake can then be used for analytics. A data lake can store both structured and unstructured data, unlike a data warehouse, which typically only stores structured data. You have much more flexibility with a data lake than a data warehouse, because you can retrieve the data in any format used by your consuming services. Data lakes are ideal to use with  Cloud Security Analytics, since Sumo Logic's parsing and mapping capabilities can transform data quickly and at scale. 

Reviewer: ```suggestion
A data lake is a centralized repository to store data. Data in the lake can then be used for analytics. A data lake can store both structured and unstructured data, unlike a data warehouse, which typically only stores structured data. Data lakes are more flexible than data warehouse in that you can retrieve the data in any format used by your consuming services.

Data lakes are ideal to use with Sumo Logic's Cloud Security Analytics - our parsing and mapping capabilities can transform your data quickly and at scale.
```

#### Step 7: Analyze security data

Once you connect your data lake to Sumo Logic, you can start analyzing security data. Cloud Security Analytics is built on a security-first principle. It is designed to analyze security data with real-time threat detection, compliance reporting, and dashboards. Our cloud-native architecture unifies security events and investigations across multiple cloud platforms, including AWS, Azure, and Google Cloud Platform.

#### Step 8: Monitor your data

With security data flowing in from your data lake, you can continuously monitor and analyze your data to identify potential threats and respond proactively. Our platform is also updated with the latest security content from our Sumo Logic Threat Labs team. This team moves quickly to support customers in crisis, like when the Log4j vulnerability surfaced. They continuously monitor and analyze data to identify threats, both known and unknown.

> ## Install a data lake for use with Cloud Security Analytics

Reviewer: It seems like this should be higher up in the doc

Author: Good point. For now, we'll leave it as-is.
